### PR TITLE
PCA svd-based description

### PR DESCRIPTION
**Pull request by mchernov-intel**
image

### CODE REVIEW

**Comment by mchernov-intel** [comment]
@outoftardis @andreynv @rlnx please, review

File: source/elements/oneDAL/source/algorithms/decomposition/pca.rst [hunk 1]
```diff
@@ -37,7 +37,28 @@ Covariance-based method
...
 ----------------
-[TBD]
+
+This method uses singular decomposition of the dataset to find its principal
+components. The following steps are distinguished:
+
+#. Singular values and singular vector computation
+#. Resulting matrix formation
+
+To compute singular values and singular vectors the following equation system
```
Comment (outoftardis) [inline source/elements/oneDAL/source/algorithms/decomposition/pca.rst:47]: Comma after vectors

File: source/elements/oneDAL/source/algorithms/decomposition/pca.rst [hunk 2]
```diff
@@ -37,7 +37,28 @@ Covariance-based method
...
+      X v = \lambda u, \\
+      X^Tu = \lambda v
+   \end{cases}
+
+where :math:`\lambda_i` is the singular value, :math:`u_i` and :math:`v_i` are
+the corresponding left and right singular vectors.
+
+The final step is to sort the set of pairs :math:`(\lambda_i, v_i)` in
+the descending order by :math:`\lambda_i` and form the resulting matrix :math:`T
+= (v_{i,1}, \cdots, v_{i,r}), \quad 1 \leq i \leq p`
```
Comment (outoftardis) [inline source/elements/oneDAL/source/algorithms/decomposition/pca.rst:61]: Full stop in the end

File: source/elements/oneDAL/source/algorithms/decomposition/pca.rst [hunk 3]
```diff
@@ -37,7 +37,28 @@ Covariance-based method
 ----------------
 SVD-based method
 ----------------
-[TBD]
+
+This method uses singular decomposition of the dataset to find its principal
```
Comment (outoftardis) [inline source/elements/oneDAL/source/algorithms/decomposition/pca.rst:41]: I think "value" is missing

File: source/elements/oneDAL/source/algorithms/decomposition/pca.rst (hunk 1 above)
Comment (mchernov-intel) [reply source/elements/oneDAL/source/algorithms/decomposition/pca.rst]: fixed

File: source/elements/oneDAL/source/algorithms/decomposition/pca.rst (hunk 2 above)
Comment (mchernov-intel) [reply source/elements/oneDAL/source/algorithms/decomposition/pca.rst]: fixed

File: source/elements/oneDAL/source/algorithms/decomposition/pca.rst (hunk 3 above)
Comment (mchernov-intel) [reply source/elements/oneDAL/source/algorithms/decomposition/pca.rst]: fixed

**Comment by mchernov-intel** [comment]
@outoftardis all comments were fixed.

File: source/elements/oneDAL/source/algorithms/decomposition/pca.rst [hunk 4]
```diff
@@ -37,7 +37,28 @@ Covariance-based method
 ----------------
 SVD-based method
 ----------------
-[TBD]
+
+This method uses singular value decomposition of the dataset to find its
+principal components. The following steps are distinguished:
```
Comment (andreynv) [inline source/elements/oneDAL/source/algorithms/decomposition/pca.rst:42]: See comments to the covariance method

File: source/elements/oneDAL/source/algorithms/decomposition/pca.rst [hunk 5]
```diff
@@ -37,7 +37,28 @@ Covariance-based method
...
+
+#. Singular values and singular vector computation
+#. Resulting matrix formation
+
+To compute singular values and singular vectors, the following equation system
+shall be resolved:
+
+.. math::
+   \begin{cases}
+      X v = \lambda u, \\
```
Comment (andreynv) [inline source/elements/oneDAL/source/algorithms/decomposition/pca.rst:52]: Do not discuss svd, just refer to the bibliography containing description of the svd methods. Implementation is free to choose any svd algorithm
The same generally applies to eigenvalue decomposition in covariance method

File: source/elements/oneDAL/source/algorithms/decomposition/pca.rst (hunk 4 above)
Comment (mchernov-intel) [reply source/elements/oneDAL/source/algorithms/decomposition/pca.rst]: fixed

File: source/elements/oneDAL/source/algorithms/decomposition/pca.rst (hunk 5 above)
Comment (mchernov-intel) [reply source/elements/oneDAL/source/algorithms/decomposition/pca.rst]: fixed

**Review by andreynv — APPROVED** [review]

**Pull request merged**